### DISCUSSION TITLE
Rebooting textadventures, Quest and Squiffy

### DISCUSSION THREAD

**Discussion by alexwarren** [textadventures/textadventures.co.uk#10] [Announcements]
![image](https://github.com/user-attachments/assets/18c7dc08-64c8-40ee-bda2-b9e6b62a0237)

I am back.

I am Alex Warren, the original creator of the textadventures.co.uk site, and the Quest 5 and Squiffy text adventure systems.

I [handed over](https://blog.textadventures.co.uk/2017/01/11/meet-the-new-textadventures-co-uk-team/) all of these in 2017, and I am very grateful to Luis and Andy for keeping things running for the last few years.

Recently I noticed that a few things around the site had stopped working, so I quietly dropped in to [fix a few issues with Squiffy](https://textadventures.co.uk/forum/squiffy/topic/zefgfs4lqkq3arknwjhqkg/various-squiffy-issues-are-now-fixed).

Well, things kind of escalated from there, and it turned out now was a good time to transfer ownership of things again. So I’m happy to announce that I am back in charge of textadventures.co.uk, Quest 5 and Squiffy once more.

There’s a whole bunch of things I want to do to revive and modernise things, and I’ve made a good start on each of them…

- rebuilding (and maybe rebranding) the textadventures.co.uk site.
- a major new version of “Quest 5”, using Blazor and WASM to make things work super fast, in browsers and across platforms. This will need a new name (as there’s already a [Quest 6](https://github.com/ThePix/QuestJS)), so I am referring to this new version as “Quest Viva”.
- a major new version Squiffy, built with TypeScript and with the editor implemented as a [PWA](https://web.dev/explore/progressive-web-apps).

Work is well underway on all of these, and I’ll announce more details as they come closer to fruition.

In the meantime, I have opened up GitHub discussions for [textadventures](https://github.com/orgs/textadventures/discussions), [Quest](https://github.com/textadventures/quest/discussions) and [Squiffy](https://github.com/textadventures/squiffy/discussions), and there’s a new [official Discord Server](https://discord.gg/vME4fcdrEp) you can join. Hope to see you there!

**Comment by DraigNET** [comment]
Welcome back Alex. Not sure anybody else is still around from Quest 3 era (Remember CyPet?), but I check in every now and then.

Any plans to make QuestNET again?

**Reply by alexwarren** [reply]
That's a proper blast from the past! I've got no current plans to make another multi-player version of Quest, but only because "current plans" is already quite a big list of projects to fill my spare time with. Who knows, maybe one day...

**Comment by Jhon09000** [comment]
Really enjoying the ideas shared here   text-based games still have so much creative potential. I recently found [ennot.us](https://ennot.us/), which has a few tools that might be helpful for devs or writers working on projects like these."

**Reply by DraigNET** [reply]
That's just a short URL thing, there's no tools.

**Reply by alexwarren** [reply]
Spam reported, for some reason I can't edit or delete, sigh.
